docs/release_process.md

```diff
@@ -0,0 +1,80 @@
+# Release Process
+
+So you want to release a new version, great!
+
+First, make sure all the desired pull requests have been merged.
+
+Next, pull [the latest translations](translate.md).
```

Comment:
We don't have to just pull the latest translations. We have to push them to transiflex, be sure they are translated and pull them.

All those steps are explain in the corresponding section, but let be more clear.